corehq/motech/fhir/models.py

```diff
@@ -31,7 +31,10 @@ class FHIRResourceType(models.Model):
 
     # `template` offers a way to define a FHIR resource if it cannot be
     # built using only mapped case properties.
-    template = JSONField(null=True, blank=True, default=None)
+    template = JSONField(default=dict)
+
+    class Meta:
+        unique_together = ('case_type', 'fhir_version')
```

Comment:
Should not this be "name, case_type, fhir_version"?

Also, if we are going to support a case type to populate information for different resources, wont this block that?
Here is my statement from the skype chat,
```
should we still have only one resource type set for a case type for search as a "case-type-level-mapping" but still let different case properties be mapped to different resource types (I believe should be feasible with some tweak to how FHIR models).
```

Reply:
> Should not this be "name, case_type, fhir_version"?

I think that depends on whether we use ValueSource to get values from supercases / subcases, or whether we use case-type-resource-type mappings.

I'll leave this until that discussion has reached a conclusion.